dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -84,7 +85,80 @@ public Pattern getParameterPattern() {
 
     public Map<String, DbParameterAccessor> getAllProcedureParameters(
             String procName) throws SQLException {
-        throw new UnsupportedOperationException();
+        String[] qualifiers = procName.split("\\.");
+        String schemaName, procedureName;
+        if (qualifiers.length == 2) {
+            schemaName = qualifiers[0];
+            procedureName = qualifiers[1];
+        } else {
+            schemaName = getConnection().getSchema();
+            procedureName = qualifiers[0];
+        }
+        procedureName = procedureName.toUpperCase();
+        DatabaseMetaData dbmd = getConnection().getMetaData();
+        ResultSet rs;
+        rs = dbmd.getProcedures(null, schemaName, procedureName);
```

Comment:
It appears that this is possible in Derby. So we could pick the wrong one. May be in other DB types too.

We have no way of knowing what type the `Execute Procedure` fixture is expecting with the current code base.